packages/dashboard/src/components/pages/explore/explore.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { Text } from '@deriv/components';
+import TempButtons from 'Components/temp-buttons';
+
+const Explore: React.FC = () => {
+    return (
+        <div className='dw-explore'>
+            <Text as='p' size='l' color='less-prominent' align='center'>
+                Explore
```

Comment:
I noticed sometimes we don't use `localize` in this PR. Any reason behind this?

Reply:
Yes, these are example pages set to be removed once development on wallet begins.